webapp/.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 .npminstall
+dist
```

Comment:
@mickmister Actually this isn't mandatory, as the top-level `.gitignore` takes care of this folder. I added it here too so that tools [like this](https://github.com/ggreer/the_silver_searcher) doesn't show matches from `dist` when run from within the `webapp` directory.

Reply:
I would like to keep `.gitignore` consistent with the other plugin repos. Would you mind reverting these change?

Reply:
Sure :+1: